README.md

```diff
@@ -45,6 +43,18 @@ Today the core uses of HELICS are in the energy domain, where there is extensive
 
 Previous and existing use cases have stretched across a wide range of scales in time and spatial area, from transient dynamics to long-term planning studies, and from individual appliance behavior to nation-wide simulations.
 
+## Philosophy of HELICS
+
+The design and development of HELICS is driven by a number of philosophical considerations that have a clear path to design decisions in the code
+
+- Make it as easy as possible for federates of all kinds to work together
+- Federates cannot impose restrictions or requirements on other federates
+- Federates should maintain control and autonomy
+- The design should be layered and modular to be adaptable to a wide variety of circumstances
+- Centralized control should be minimized
+
+These design priorities directed much of the design of HELICS and supporting tools, including operation as a library vs RTI, the use of distributed timing and control, and giving federates fine grained control over the time management and operations. These support an underlying belief driving co-simulation that "Simulations are Better Together"
```

Comment:
RTI should be defined -- I don't think the average reader coming across HELICS will know what it is (I'm assuming it is referring to run-time infrastructure operating as part of HLA).
```suggestion
## Philosophy of HELICS

The design and development of HELICS is driven by a number of philosophical considerations that have a clear path to design decisions in the code

- Make it as easy as possible for federates of all kinds to work together
- Federates cannot impose restrictions or requirements on other federates
- Federates should maintain control and autonomy
- The design should be layered and modular to be adaptable to a wide variety of circumstances
- Centralized control should be minimized

These design priorities directed much of the design of HELICS and supporting tools, including operation as a library rather than component for HLA (High Level Architecture), the use of distributed timing and control, and giving federates fine grained control over the time management and operations. These support an underlying belief driving co-simulation that "Simulations are Better Together"
```

Reply:
We should probably spell it out more but it isn't just HLA, FMI, Mosaik, and several others all use RTI,  basically the difference between HELICS being the executable main and loading a program vs, a program being the main and loading HELICS.

Reply:
Something like `vs simulators being HELICS plugins`?

Reply:
I updated this in the lasted commit, hopefully it is a little clearer